xml/networking-octavia_admin.xml

```diff
@@ -403,6 +403,145 @@ Updated port: 0b0301b9-4ee8-4fb6-a47c-2690594173f4</screen>
    </para>
   </warning>
  </section>
+ <section xml:id="octavia-admin-delete">
+   <title>Removing load balancers</title>
+   <para>
+     You may find that you are unable to delete load balancers that are not
+     in the <literal>ACTIVE</literal> state. If you have load balancers that
+     cannot be deleted but are in the <literal>ERROR</literal> or
+     <literal>Pending_delete</literal> state, follow these directions when you
+     want to delete them. To clean up the bad instances run the
+     following MySQL commands manually.
+   </para>
+   <para>
```

Comment:
It's failing because of this <para> and then the next line is <procedure> - so all you need to do is remove this line. That's geekodoc for you, so building it you'll be running the validation against docbook, not geekodoc.